docs/how_to/languages.rst

```diff
@@ -0,0 +1,123 @@
+.. _multilingual_support_how_to:
+
+###############################
+How to serve multiple languages
+###############################
+
+If you used the `django CMS installer <https://github.com/nephila/djangocms-installer>`_ to start your project, you'll find
+that it's already set up for serving multilingual content. Our :ref:`installation` guide also does the same.
+
+Thiss guide specifically describes the steps required to set up multilingual support.
```

Comment:
I think "specifically" is not necessary here.
"This guide describes the steps.."